examples/fineFoods/admin/mui/src/components/product/categoryFilter.tsx

```diff
@@ -0,0 +1,103 @@
+import { useState, useEffect } from "react";
+import { LoadingButton } from "@mui/lab";
+import { CrudFilters, useList, useTranslate } from "@pankod/refine-core";
+import { Stack, Grid } from "@pankod/refine-mui";
+
+import { ICategory } from "interfaces";
+
+type ProductItemProps = {
+    setFilters?: (filters: CrudFilters) => void;
+    filters: CrudFilters;
+    categoryData: ICategory[];
+    isLoading: boolean;
+};
+
+export const CategoryFilter: React.FC<ProductItemProps> = ({
+    setFilters,
+    filters,
+    categoryData,
+    isLoading,
+}) => {
+    const t = useTranslate();
+
+    const [filterCategories, setFilterCategories] = useState<string[]>([]);
+
+    useEffect(() => {
+        setFilters?.([
+            { ...filters[0] },
```

Comment:
```suggestion
            { ...filters[0] },
```
why are we doing this?